src/app/form/(formSteps)/components/DoulaAddress.tsx

```diff
@@ -88,7 +88,7 @@ export const DoulaAddress = <T extends FieldValues>(props: DoulaAddressProps<T>)
           >
             {Object.keys(AddressState).map((state) => (
               <option key={state} value={state}>
-                {state}
+                {addressStateToName[state as keyof typeof AddressState]}
               </option>
             ))}
           </Select>
```

Comment:
[dust] if you move to the data structure you mentioned below

```
export enum AddressState {
  AL = "Alabama",
 ...
```

could that make this part more like this? the `as keyof typeof` gives me pause

```
{Object.entries(AddressState).map(([stateCode, stateName]) => (
  <option key={stateCode} value={stateCode}>
    {stateName}
  </option>
))}
```